# Description: Fix issue where writable option is wrongly detected as false
# Origin: vendor
# Bug-Ubuntu: https://bugs.launchpad.net/bugs/280480
# Forwarded: no
# Author: Chow Loong Jin <hyperair@ubuntu.com>
---

 src/shares.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)


diff --git a/src/shares.c b/src/shares.c
index ddec01b..f65f78d 100644
--- a/src/shares.c
+++ b/src/shares.c
@@ -358,9 +358,9 @@ add_key_group_to_hashes (GKeyFile *key_file, const char *group)
 
 	acl = get_string_from_key_file (key_file, group, KEY_ACL);
 	if (acl) {
-		if (strcmp (acl, "Everyone:R") == 0)
+		if (strstr (acl, "Everyone:R"))
 			is_writable = FALSE;
-		else if (strcmp (acl, "Everyone:F") == 0)
+		else if (strstr (acl, "Everyone:F"))
 			is_writable = TRUE;
 		else {
 			g_message ("unknown format for key '%s/%s' as it contains '%s'.  Assuming that the share is read-only",
